docs/DeveloperGuide.md

```diff
@@ -29,6 +29,30 @@ a Command Line Interface (CLI).
 
 
 ### 1.5 Overview of Application
+In order to better understand how DonnaFin helps financial advisors, it is useful to understand and model the status
+quo without DonnaFin.
+
+Below is an Object-Oriented Domain Model (OODM) modelling the existing state of affairs between a Financial Advisor and
+their Client/s.
+
+![OverviewOodmDiagram](images/OverviewOodmDiagram.png)
+
+We can see that Financial Advisors generally have one or more clients that they have to manage, and
+every client has the following unique information that is important for the Financial Advisors to keep track of:
+take note of:
+1. `Contact Details`
+2. `Policies`
+3. `Assets`
+4. `Liabilities`
+5. `Notes`
```

Comment:
* Financial Advisors are not proper nouns, should be all lower case
* I believe the client has the "following categories of information unique to them" is more appropriate to these listed categories of info.
* the listed categories of information should not be in code form, just normal text as they are normal, non-code category names.